pkg/flink/resources.go

```diff
@@ -201,6 +201,8 @@ func NewFlinkCluster(config *Config, taskCtx FlinkTaskContext) (*flinkOp.FlinkCl
 		cluster.Spec.Image.Name = image
 	}
 
+	cluster.Spec.FlinkVersion = config.DefaultFlinkCluster.Spec.FlinkVersion
```

Comment:
There's no need for this. FlinkVersion is already being set through config.

Reply:
hmm, could you link to the code where we set this property?

Reply:
https://github.com/spotify/flyte-flink-plugin/blob/f5602c96e943a25ef3d93977a701ac3266f8759b/pkg/flink/resources.go#L181